packages/cli/vite.config.unit.ts

```diff
@@ -10,12 +10,37 @@ export default defineConfig({
     cache: {
       dir: '../../node_modules/.vitest',
     },
+    alias: [
+      {
+        find: '@code-pushup/test-utils',
+        replacement: new URL('../../testing/test-utils/src', import.meta.url)
+          .pathname,
+      },
+      {
+        find: '@code-pushup/core',
+        replacement: new URL('../core/src', import.meta.url).pathname,
+      },
+      {
+        find: '@code-pushup/models',
+        replacement: new URL('../models/src', import.meta.url).pathname,
+      },
+      {
+        find: '@code-pushup/utils',
+        replacement: new URL('../utils/src', import.meta.url).pathname,
+      },
+    ],
```

Comment:
Unfortunately, I found no alternative to make path resolution work.
I tried :
- to remove/restore `nxViteTsPaths` from plugins
- to configure `viteTsConfigPaths ` in many ways in the plugins
- to configure `resolve.preserveSymlinks = true` 

Here are some references I followed:
- https://vitest.dev/guide/migration.html https://github.com/vitest-dev/vitest/pull/3961 https://vitejs.dev/guide/dep-pre-bundling.html#monorepos-and-linked-dependencies
- https://github.com/vitejs/vite/discussions/14172